src/style/colors.ts

```diff
@@ -143,6 +147,24 @@ export class Colors {
     return this.getTintedColorForDynamicHex(color, tintKey);
   }
 
+  getInvertedTintKey(tintKey: string | number) {
```

Comment:
can I offer a code improvement (: 

You can create an array of our keys (`[1, 5, 10, 20, 30, ..., 80]`) 
Then use `array.indexOf` to detect the given key index, revert the index (as you did before) and return the inverted key..

Reply:
done :)